### PR TITLE
Feat css

### PR DESCRIPTION
**Pull request by emlindvall**
____ Wrote Tests _x__ Implemented ____ Reviewed

## Neccesary checkmarks
- [ ] All Tests are Passing in all environments
- [x] The code will run locally
## Type of change
- [x] New feature
- [ ] Bug Fix
- [ ] Refactor

## Description of Change:

  description: Adds CSS styling for TruckCard component,  implements conditional styling for the currentUser toggle button, adds additional pin logo asset.
    
## Requested feedback: 
    
  I'd like feedback on... 

## Check the correct boxes

- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes

- [ ] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

- [ ] My code has no unused/commented out code
- [ ] My code has no binding.pry's
- [ ] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

### CODE REVIEW

File: src/Components/Header/Header.js [hunk 1]
```diff
@@ -2,16 +2,16 @@ import React, { useEffect, useState } from 'react';
...
   return (
     <div className="header-container">
       <div className="logo-container">
         <img src={logo} id="spin-hover" className="logo-img" alt="streeteatz-logo"/>
         <p className="logo-name">street eatz</p>
       </div>
       <div className="toggle-container">
-        <button id="vendor" className="vendor-btn btn" onClick={(event) => togView(event.target.id)}>vendors</button>
-        <button id="customer" className="customer-btn selected-btn" onClick={(event) => togView(event.target.id)}>customers</button>
+        <button id="vendor" className={ `${currentUser === "vendor" ? "selected-btn" : "btn"}`} onClick={(event) => togView(event.target.id)}>vendors</button>
```
Comment (Devynnem) [inline src/Components/Header/Header.js:13]: This is really cool!

File: src/Components/TruckCard/TruckCard.js [hunk 2]
```diff
@@ -52,37 +54,47 @@ const currentUpvote = isUpvoted ? upvoted : notUpvoted
...
+
+    <div className="details-container">
+      <p className="distance">0.4 miles</p>
+
+      <div className="status-container">
+        <p className="status">{props.truck.status}</p>
+        <img className="pin" src={pin}></img>
+        <p className="location">{props.truck.location}</p>
+          {/* we're going to need to figure out what to do with the props.truck.location coordinates to make them display in a way that's like, readable for a human being */}
+          {/* we're also going to need to figure out how to do a distance calculation based on lat/long coordinates */}
```
Comment (Devynnem) [inline src/Components/TruckCard/TruckCard.js:75]: Yeah, this will be interesting to come up wit

**Review by Devynnem — APPROVED** [review]

File: src/Components/Search/Search.css [hunk 3]
```diff
@@ -85,6 +85,12 @@
   border: 1.5px #767676 solid;
   border-radius: 25px;
   cursor: pointer;
+  transition: all .3s ease-in-out;
+}
+
+.filter-btn:hover {
+  width: 100px;
+  transition: all .3s ease-in-out;
```
Comment (Pma913) [inline src/Components/Search/Search.css:93]: Can't wait to see these transition effects!

**Review by Pma913 — APPROVED** [review]
This all looks great. Thank you

**Pull request merged**